### PR TITLE
Improve Scaffold.drawer and Navigator.popUntil docs

### LINKED ISSUE #6963: Navigator.popUntil docs should mention ModalRounte.withName
A common pattern is to want to pop until a certain route name.  We have code for this, but there's no breadcrumb.

### PR DESCRIPTION
**Pull request by abarth**
Fixes #6963

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/material/scaffold.dart [hunk 1]
```diff
@@ -358,15 +368,23 @@ class Scaffold extends StatefulWidget {
...
+  /// [ScaffoldState.openDrawer] function.
   ///
   /// Typically a [Drawer].
   final Widget drawer;
 
   /// A bottom navigation bar to display at the bottom of the scaffold.
   ///
-  /// Snack bars slide from underneath the botton navigation while bottom sheets
-  /// are stacked on top.
+  /// Snack bars slide from underneath the bottomm navigation bar while bottom
```
Comment (Hixie) [inline packages/flutter/lib/src/material/scaffold.dart]: bottom

File: packages/flutter/lib/src/material/scaffold.dart [hunk 2]
```diff
@@ -624,10 +646,13 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   /// Returns a contoller that can be used to close and otherwise manipulate the
   /// button sheet.
   ///
+  /// See [Scaffold.of] for information about how to obtain the [ScaffoldState].
```
Comment (Hixie) [inline packages/flutter/lib/src/material/scaffold.dart]: move this into the bulletted list, since otherwise it reads weird

**Comment by Hixie** [comment]
LGTM

**Pull request merged**